### DISCUSSION TITLE
Parse data in flutter eval

### DISCUSSION THREAD

**Discussion by rithamto** [ethanblake4/dart_eval#110] [Q&A]
![image](https://github.com/ethanblake4/dart_eval/assets/66898326/27f3cdca-8ddd-4fda-b5b2-d4956f88c121)

i already got the string json from http(in image). But can't parse string to json. Pls help

**Comment by ethanblake4** [comment]
Hi, v0.6.1 fixes an issue with accessing the data from json.decode() in dart_eval, please try it and let me know if you are still having problems.
